src/components/SignalCard/SignalCard.js

```diff
@@ -1,12 +1,15 @@
-import React, { Fragment, useState } from 'react'
+import React, { Fragment } from 'react'
 import { Link } from 'react-router-dom'
 import cx from 'classnames'
 import Panel from '@santiment-network/ui/Panel/Panel'
 import Icon from '@santiment-network/ui/Icon'
 import Toggle from '@santiment-network/ui/Toggle'
+import Button from '@santiment-network/ui/Button'
+import { DesktopOnly, MobileOnly } from './../../components/Responsive'
 import MultilineText from '../../components/MultilineText/MultilineText'
 import StatusLabel from './../../components/StatusLabel'
 import { RemoveSignalButton, SignalTypeIcon } from './controls/SignalControls'
+import ContextMenu from '@santiment-network/ui/ContextMenu'
```

Comment:
wrong order( 
`ContextMenu` should be above

Reply:
Fill wix